CMakeLists.txt

```diff
@@ -89,9 +89,15 @@ else ()
   #   interfaces, etc.
   # disable -Wpointer-arith: this is a GCC extension, and doesn't work in MSVC.
   add_definitions(
-    -Wall -Wextra -Werror -Wno-unused-parameter -Wpointer-arith -g
+    -Wall -Wextra -Werror -Wno-unused-parameter -Wpointer-arith -g -std=c++11
```

Comment:
I sympathize, but our machines are pretty outdated at this point (coming up on a 3-year-old distro of a 4-year-old compiler branch, and replacement LTS has been out for nearly a year now). It seems unlikely we'd make such concessions for Windows or Mac users, so why should we do that for ourselves? It's really easy to build a daily-driver clang or gcc (libc++ is a little harder, but still not that hard).